From edff537be5baf8955c4f22abf00319280623d1ca Mon Sep 17 00:00:00 2001
From: Stuart Menefy <stuart.menefy@codasip.com>
Date: Wed, 10 Jan 2024 18:41:10 +0000
Subject: [PATCH 04/17] hw/riscv/hobgoblin: Change PLIC "priority-base"

Commit 5decd2c52 ("hw/intc: sifive_plic: Change "priority-base" to
start from interrupt source 0") changed the offset where the
priority register start. Make the same change for the Hobgoblin
platform.

Signed-off-by: Stuart Menefy <stuart.menefy@codasip.com>
---
 hw/riscv/hobgoblin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/riscv/hobgoblin.c b/hw/riscv/hobgoblin.c
index 198572eb59..86827a0f97 100644
--- a/hw/riscv/hobgoblin.c
+++ b/hw/riscv/hobgoblin.c
@@ -58,7 +58,7 @@ static const struct MemmapEntry {
 #define HOBGOBLIN_PLIC_HART_CONFIG "M" /* Not "MS" unfortunatly */
 #define HOBGOBLIN_PLIC_NUM_SOURCES 32
 #define HOBGOBLIN_PLIC_NUM_PRIORITIES 7
-#define HOBGOBLIN_PLIC_PRIORITY_BASE 0x04
+#define HOBGOBLIN_PLIC_PRIORITY_BASE 0x0000
 #define HOBGOBLIN_PLIC_PENDING_BASE 0x1000
 #define HOBGOBLIN_PLIC_ENABLE_BASE 0x2000
 #define HOBGOBLIN_PLIC_ENABLE_STRIDE 0x80
-- 
2.39.3

